src/main/scala/com/microsoft/hyperspace/actions/OptimizeAction.scala

```diff
@@ -106,20 +105,39 @@ class OptimizeAction(
 
     if (filesToOptimize.isEmpty) {
       throw NoChangesException(
-        "Optimize aborted as no index files smaller than " +
+        "Optimize aborted as no applicable files smaller than " +
```

Comment:
Can you revise this message and make it more clear? (I am not sure "applicable" is an easy-to-know word in the context of optimize).

Reply:
How about  "no optimizable index files"?

Reply:
Yes, that is precise and much better. Thanks!